cmake/projects/binaryen/hunter.cmake

```diff
@@ -9,22 +9,34 @@ include(hunter_download)
 include(hunter_pick_scheme)
 include(hunter_cmake_args)
 
+
+hunter_add_version(
+    PACKAGE_NAME
+    binaryen
+    VERSION
+    1.38.28-p0
```

Comment:
Why do you put it here when it was not used and we already have p1?

Reply:
Were trying to revert binary, as we found out that new versions have completely unfeasible performance, but it turned out there are weird discrepancies between soramitsu/binaryen and Warchant/binaryen repos. Better deal with it in a separate PR.